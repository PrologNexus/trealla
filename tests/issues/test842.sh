#!/bin/sh

TMP=tmp.out
TMPPL=tmp.pl

trap "rm -f $TMPPL" EXIT

cat >$TMPPL <<EOF
vx(X) :- -X=X.
EOF

$TPL $TMPPL <<EOF >$TMP
vx(Y), vx(Z), X = 1.
halt.
EOF

rm $TMPPL
