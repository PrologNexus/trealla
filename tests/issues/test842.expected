   Y = -_Y, X = 1.
